无法打包构建，报 ts 的 png 类型提示错误

具体报错信息：
```bash
...
src/MyTestImagePack/index.tsx(4,19): error TS2307: Cannot find module './example-light.png' or its corresponding type declarations.
...
```

--- a/src/MyTestImagePack/index.tsx
+++ b/src/MyTestImagePack/index.tsx
@@ -1,6 +1,7 @@
 import React from 'react';
 
 import './index.less';
+// @ts-ignore
 import light from './example-light.png';
 
 class TestImagePack extends React.Component {

